### DISCUSSION TITLE
12ft.io

### DISCUSSION THREAD

**Discussion by MikaelaArsenault** [vercel/community#4659] [Help] [answered]
### Summary

Does anyone know why 12ft.io is down? When I try to open it, I get the following message:

[
![](https://i.postimg.cc/dtWxT12Y/IMG-0756.jpg)
](url)

### Steps to Reproduce

Clearing the site from my Safari history

**Comment by swarnava** [comment, marked as answer]
Hi,

We don't provide specific information about website data in this forum.
